x/ccv/provider/client/cli/query.go

```diff
@@ -78,7 +78,7 @@ func CmdConsumerChains() *cobra.Command {
 		Short: "Query consumer chains for provider chain.",
 		Long: `Query consumer chains for provider chain. An optional
 		integer parameter can be passed for phase filtering of consumer chains,
-		(Registered=1|Initialized=2|Launched=3|Stopped=4).`,
+		(All=0|Registered=1|Initialized=2|Launched=3|Stopped=4).`,
```

Comment:
isn't 'All' redundant as this is the default (no filtering) functionality.
To me this gives the expression that the default result  (no filter option given) is not returning
alll consumer chains.

Reply:
I agree. I'll remove All.